src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -174,9 +191,12 @@ export const InitialCashoutView = ({
                 }
             }
             onNext()
-        } catch (error) {
+        } catch (error: any) {
             console.error('Error:', error)
-            setErrorState({ showError: true, errorMessage: 'An error occurred. Please try again.' })
+            setErrorState({
+                showError: true,
+                errorMessage: error.message || 'An error occurred. Please try again.',
+            })
```

Comment:
intention: this propagates the error message like insufficient funds etc. Much more informative.